source/quickstart/macos.rst

```diff
@@ -87,26 +101,33 @@ Procedure
 
 #. Connect to the Server
 
-   In your browser, go to http://127.0.0.1:9000.
+   Open http://127.0.0.1:9000 in a web browser to access the :ref:`MinIO Console <minio-console>`. 
+   Alternatively, you can enter any of the network addresses specified as part of the server command output.
 
-   Access the :ref:`minio-console` by going to a browser (such as Safari) and entering one of the Console address specified in the :mc:`minio server` command's output.
+   Access the :ref:`minio-console` by going to a browser (such as Safari) and enter one of the Console address specified in the :mc:`minio server` command's output.
```

Comment:
These blocks are saying the same thing - we should keep one of them. I smushed these together in my linux PR, if you want to crib/improve on that.